### DISCUSSION TITLE
[Esame 17 aprile 2024] Esercizio 3

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#460] [Esami M-Z]
![17 apr es.3](https://github.com/user-attachments/assets/5cc9af80-791e-487c-a922-8ceccf0beaa5)

**Comment by thegeek-sys** [comment]
![immagine](https://github.com/user-attachments/assets/4a21928f-3663-4b42-8121-73f889fc426b)

**Comment by niccolo-fato** [comment]
<img width="670" height="1280" alt="image" src="https://github.com/user-attachments/assets/7d29c7c3-ef9d-4313-833c-f8323715c92f" />
